android/app/src/main/java/org/smartregister/fhircore/fragment/PatientListFragment.kt

```diff
@@ -111,9 +119,49 @@ class PatientListFragment : Fragment() {
       )
 
     patientListViewModel.searchResults(page = 0, pageSize = pageCount)
+      setUpBarcodeScanner()
     super.onViewCreated(view, savedInstanceState)
   }
 
+    private fun setUpBarcodeScanner() {
```

Comment:
can we add tests for these funs?

Reply:
Fragment testing and mocking are not yet set up for this repo. I've done a bit of experimenting and it looks like I'll need to put in more time to figure this out (encountered non-trivial errors). This PR is not targeted to main and was meant for the covax-demo branch, do I dedicate more time into this?

Reply:
no